modules/ROOT/pages/networking.adoc

```diff
@@ -153,6 +153,11 @@ Configure a remote server to run an {ocp} cluster provided by {prod}.
 This procedure assumes the use of a {rhel}, {fed}, or {centos} server.
 Run every command in this procedure on the remote server.
 
+[NOTE]
+====
+These instructions will only work for system-mode networking.
+====
+
```

Comment:
We should have section around networking mode so user get to know how to change networking mode.

Reply:
What was supposed to be #26, as this was a 'quick-fix'. Didn't turn out to be the quick fix it was supposed to be.